app/gpg.go

```diff
@@ -0,0 +1,115 @@
+package app
+
+import (
+	"bytes"
+	"encoding/base64"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"golang.org/x/crypto/openpgp/armor"
+	"golang.org/x/crypto/openpgp/packet"
+)
+
+func FormatPubKey(pubkeyFilePath string) (string, error) {
+	pubKey, err := ioutil.ReadFile(pubkeyFilePath)
+
+	if err != nil {
+		return "Error opening File: ", err
+	}
+
+	if strings.Contains(string(pubKey), "-----BEGIN PGP") {
+		b, err := os.Open(pubkeyFilePath)
```

Comment:
you're reading the file twice here; perhaps remove the call on line 22 and just reuse `pubKey` since you've already read all the bytes in ?

Reply:
agree, well spotted.